geode-core/src/distributedTest/java/org/apache/geode/distributed/internal/P2PMessagingConcurrencyDUnitTest.java

```diff
@@ -194,6 +193,7 @@ public void testP2PMessaging(
           throw new RuntimeException("doSending failed", e);
         }
         final int firstMessageId = senderId * SENDER_COUNT;
+        final Random random = new Random(RANDOM_SEED);
```

Comment:
How did plain old `Random` slip back into this test?

Reply:
oh: by moving it into the lambda, it's no longer shared across threads. And by constructing a `Random` I retain a scrap of determinism, by being able to seed it.

Reply:
> And by constructing a `Random` I retain a scrap of determinism, by being able to seed it.

Oxymoron. ;)